fastlane/lib/fastlane/actions/install_on_device.rb

```diff
@@ -11,7 +11,7 @@ def self.run(params)
           "ios-deploy",
           params[:extra],
           "--bundle",
-          params[:ipa]
+          params[:ipa].gsub(/ /, '\ ')
```

Comment:
`ipa` param seems to be optional.

So you have to check nil.

https://github.com/fastlane/fastlane/pull/8582/files#diff-cec084ffdd8e2b1d290fbc3ba9bacb06R55